admin/src/js/controllers/edit-user.js

```diff
@@ -371,9 +435,8 @@ angular
     };
 
     const computeFields = () => {
-      $scope.editUserModel.place = $(
-        '#edit-user-profile [name=facilitySelect]'
-      ).val();
+      const placeValue = $('#edit-user-profile [name=facilitySelect]').val();
+      $scope.editUserModel.place = Array.isArray(placeValue) && placeValue.length === 0 ? null : placeValue;
```

Comment:
I'm not sure id this works as expected. 
If `placeValue` is a string, than this will return a string. And in lots of places we assume this will be an array.

Reply:
I added this after realizing an admin user or certain users can have `null` for their `placeValue`. If `placeValue` is a string, then it is typecasted to an `Array`. I don't see the situation it would be a `string`.